packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache

```diff
@@ -2,7 +2,7 @@
   <section class="modal-panel">
     <div class="main-avatar">
       <div id="loading-avatar-spinner">
-        <img alt="" src="../../../images/spinnerlight.svg">
+        <img alt="Loading..." src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iNzNweCIgaGVpZ2h0PSI3M3B4IiB2aWV3Qm94PSIwIDAgNzMgNzMiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgICA8IS0tIEdlbmVyYXRvcjogU2tldGNoIDQ2LjIgKDQ0NDk2KSAtIGh0dHA6Ly93d3cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaCAtLT4KICAgIDxkZWZzPgogICAgICAgIDxsaW5lYXJHcmFkaWVudCB4MT0iOTMuMDkyODA5NiUiIHkxPSI1Mi43NzM0Mzc1JSIgeDI9IjY4LjUxMzMzOTglIiB5Mj0iMTE5LjMyNjAwNyUiIGlkPSJsaW5lYXJHcmFkaWVudC0xIj4KICAgICAgICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzBBODRGRiIgc3RvcC1vcGFjaXR5PSIwIiBvZmZzZXQ9IjAlIj48L3N0b3A+CiAgICAgICAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMwQTg0RkYiIG9mZnNldD0iNjkuMzY5ODE4MiUiPjwvc3RvcD4KICAgICAgICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzBBODRGRiIgb2Zmc2V0PSIxMDAlIj48L3N0b3A+CiAgICAgICAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMyNDg0QzYiIHN0b3Atb3BhY2l0eT0iMC4wMDQ3Nzc2Njk1MSIgb2Zmc2V0PSIxMDAlIj48L3N0b3A+CiAgICAgICAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMyNDg0QzYiIHN0b3Atb3BhY2l0eT0iMCIgb2Zmc2V0PSIxMDAlIj48L3N0b3A+CiAgICAgICAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMyNDg0QzYiIHN0b3Atb3BhY2l0eT0iMCIgb2Zmc2V0PSIxMDAlIj48L3N0b3A+CiAgICAgICAgPC9saW5lYXJHcmFkaWVudD4KICAgICAgICA8cmVjdCBpZD0icGF0aC0yIiB4PSIwIiB5PSIwIiB3aWR0aD0iNDgiIGhlaWdodD0iNjAiPjwvcmVjdD4KICAgIDwvZGVmcz4KICAgIDxnIGlkPSJQYWdlLTEiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPgogICAgICAgIDxnIGlkPSJTaGFwZSIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTUuMDAwMDAwLCAtMS4wMDAwMDApIj4KICAgICAgICAgICAgPHBhdGggZD0iTTQxLjgsNzMuOCBDMjEuOSw3My44IDUuOCw1Ny43IDUuOCwzNy44IEM1LjgsMTguMSAyMS42LDIuMiA0MS4xLDEuOCBDNDEuMywxLjggNDEuNCwxLjggNDEuNCwxLjggQzQxLjUsMS44IDQxLjcsMS44IDQxLjgsMS44IEM0NC42LDIuMiA0Ni44LDQuNSA0Ni44LDcuMyBDNDYuOCwxMC4xIDQ0LjYsMTIuNSA0MS44LDEyLjcgQzI4LDEyLjggMTYuOCwyNCAxNi44LDM3LjggQzE2LjgsNTEuNiAyOCw2Mi44IDQxLjgsNjIuOCBDNTUuNiw2Mi44IDY2LjgsNTEuNiA2Ni44LDM3LjggTDc3LjgsMzcuOCBDNzcuOCw1Ny43IDYxLjcsNzMuOCA0MS44LDczLjggWiIgZmlsbD0idXJsKCNsaW5lYXJHcmFkaWVudC0xKSI+PC9wYXRoPgogICAgICAgICAgICA8bWFzayBpZD0ibWFzay0zIiBmaWxsPSJ3aGl0ZSI+CiAgICAgICAgICAgICAgICA8dXNlIHhsaW5rOmhyZWY9IiNwYXRoLTIiPjwvdXNlPgogICAgICAgICAgICA8L21hc2s+CiAgICAgICAgICAgIDxnIGlkPSJNYXNrIj48L2c+CiAgICAgICAgICAgIDxwYXRoIGQ9Ik00MS44LDczLjggQzIxLjksNzMuOCA1LjgsNTcuNyA1LjgsMzcuOCBDNS44LDE4LjEgMjEuNiwyLjIgNDEuMSwxLjggQzQxLjMsMS44IDQxLjQsMS44IDQxLjQsMS44IEM0MS41LDEuOCA0MS43LDEuOCA0MS44LDEuOCBDNDQuNiwyLjIgNDYuOCw0LjUgNDYuOCw3LjMgQzQ2LjgsMTAuMSA0NC42LDEyLjUgNDEuOCwxMi43IEMyOCwxMi44IDE2LjgsMjQgMTYuOCwzNy44IEMxNi44LDUxLjYgMjgsNjIuOCA0MS44LDYyLjggQzU1LjYsNjIuOCA2Ni44LDUxLjYgNjYuOCwzNy44IEw3Ny44LDM3LjggQzc3LjgsNTcuNyA2MS43LDczLjggNDEuOCw3My44IFoiIGZpbGw9IiMwQTg0RkYiIG1hc2s9InVybCgjbWFzay0zKSI+PC9wYXRoPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+Cg==">
```

Comment:
I'm not sure if you want the `Loading` text. Is it at all useful? Also if we keep it then we should probably translate it too?

Reply:
> Is it at all useful?

Of course it is, as it improves accessibility. AFAIK, screen readers use it.

> Also if we keep it then we should probably translate it too?

You're right. Let me check how we are handing i18n in the codebase.

Reply:
> I'm not sure if you want the Loading text. Is it at all useful?

Me either, however I don't think it will hurt. Lets wrap in translation block if keeping it.